### PR TITLE
wg-structured-logging: remove regular meeting

### PR DESCRIPTION
**Pull request by pohly**
Meeting participation has been low for a while now because there wasn't anything to discuss. Communication via Slack is sufficient.

/assign @serathius

### CODE REVIEW

**Comment by serathius** [comment]
From verify failure
```
Do not manually edit sig-list.md or README.md files inside the sig folders.
Instead make your changes to sigs.yaml, then run `make`, and then
commit your changes to sigs.yaml and any generated docs.
```

**Comment by pohly** [comment]
Should be fixed now (verified locally).

File: sigs.yaml [hunk 1]
```diff
@@ -3439,15 +3439,7 @@ workinggroups:
...
-  meetings:
-  - description: Regular Meeting
-    day: Thursday
-    time: "15:30"
-    tz: London-UK
-    frequency: biweekly
-    url: https://zoom.us/j/96716142646?pwd=VmgrN29sbmhDREp3R0NtZlpGSlZ4Zz09
-    archive_url: https://docs.google.com/document/d/1R9bZ34L9vR1ftH0dFeOp-j50lLh5ijKVwwXJ3LDrY4I/edit?usp=sharing
-    recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP2GY3so1z2Cnkvkt5A_x1Aw
+  meetings: []
```
Comment (serathius) [inline sigs.yaml:3442]: Should we keep the links to notes and recordings?

Comment (pohly) [reply sigs.yaml:3442]: Let's check... no, the tool doesn't like a `meetings` entry without time.

There is a `BEGIN CUSTOM CONTENT` section in README.md, but adding anything there leads to a verification failure. :shrug:

Comment (serathius) [reply sigs.yaml:3442]: Ok, lets assume people can do git history.

**Pull request merged**